File: bot-javascript/src/commands/sorte.js
const { MessageEmbed } = require('discord.js');
const { gerarNumeroRandom, cores } = require('../utils');

const execute = (bot, msg, args) => {
    const sorte = gerarNumeroRandom(1, 100);

    const resposta = new MessageEmbed()
    .setColor(cores[gerarNumeroRandom(0, 8)])
    .setTitle(`${msg.author.username}`)
    .setDescription(`Você está com ${sorte}% de sorte hoje.`)

    msg.channel.send(resposta);
};

module.exports = {
    nome: "sorte",
    descricao: "Mostra com quanta sorte você está hoje",
    execute
};

File: bot-csharp/src/BotDiscord.Application/Commands/Sorte.cs
using System;
using System.Threading.Tasks;
using BotDiscord.Application.Common;
using Discord.Commands;

namespace BotDiscord.Application.Commands
{
    public class Sorte : CommandModuleBase
    {
        public override string CommandName => "sorte";
        public override string CommandDescription => "Mostra com quanta sorte você está hoje";

        [Command("sorte")]
        public async Task SorteCommand()
        {
            var random = new Random();
            var sorte = random.Next(1, 101);

            var resposta = CustomColorEmbed()
                .WithTitle($"Fala {Context.Message.Author.Username}!")
                .WithDescription($"Você está com {sorte}% de sorte hoje.")
                .WithCurrentTimestamp()
                .Build();

            var mensagemEnviada = await SendCustomMessageAsync(resposta);
        }
    }
}